content/en/docs/concepts/architecture/cri.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Container Runtime Interface (CRI)
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+
```

Comment:
```suggestion
You need a working
{{<glossary_tooltip text="container runtime" term_id="container-runtime">}} on
each Node in your cluster, so that the
{{< glossary_tooltip text="kubelet" term_id="kubelet" >}} can launch
{{< glossary_tooltip text="Pods" term_id="pod" >}} and their containers.

```
(if accepting this, move `{{< glossary_tooltip text="kubelet" term_id="kubelet" >}}` and `{{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}` here)